syncpack.config.cjs

```diff
@@ -0,0 +1,195 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+module.exports = {
+	indent: "\t",
+
+	// Don't set dep versions based on the version of the package in the workspace
+	workspace: false,
+
+	// Custom types are used to define additional fields in package.json that contain versions that should be
+	// checked/synced. See https://github.com/JamieMason/syncpack/blob/master/README.md#customtypes for more details.
+	customTypes: {
+		enginesNpm: {
+			path: "engines.npm",
+			strategy: "version",
+		},
+		enginesNode: {
+			path: "engines.node",
+			strategy: "version",
+		},
+		packageManager: {
+			path: "packageManager",
+			strategy: "name@version",
+		},
+	},
+
+	/**
+	 * SemverGroups are used to ensure that groups of packages use the same semver range for dependencies.
+	 *
+	 * semverGroup rules are applied in order to package/dep combinations. First matching rule applies. When running
+	 * `syncpack lint-semver-ranges`, the output is grouped into numbered groups. However, the numbers of the groups are
+	 * the _inverse_ of their order in this array. That is, Semver Group 1 in the output will correspond to the _last_
+	 * rule in this array.
+	 */
+	semverGroups: [
+		// GROUP 8
+		// engines.node should always use >= range
+		{
+			dependencyTypes: ["enginesNode"],
+			dependencies: ["**"],
+			packages: ["**"],
+			range: ">=",
+		},
+
+		// GROUP 7
+		// engines.npm should always use ^ range
+		{
+			dependencyTypes: ["enginesNpm"],
+			dependencies: ["**"],
+			packages: ["**"],
+			range: "^",
+		},
+
+		// GROUP 6
+		// packageManager should always use exact version
+		{
+			dependencyTypes: ["packageManager"],
+			dependencies: ["**"],
+			packages: ["**"],
+			range: "",
+		},
+
+		// GROUP 5
+		// PropertyDDS packages' dependencies are ignored because they use a lot of exact deps.
```

Comment:
Do we have anything tracking fixing this? Seems like we should update them.

Reply:
PropertyDDS is its own little world. We have a bunch of tracking issues to update them. I think we need to have a strategy before we make changes to them.